protobuf/src/misc.rs

```diff
@@ -10,6 +12,63 @@ pub unsafe fn remove_lifetime_mut<A: ?Sized>(a: &mut A) -> &'static mut A {
     mem::transmute(a)
 }
 
+macro_rules! map_rust_type_to_protobuf {
```

Comment:
Can we do copy-paste instead of macro? Macros are not used in rust-protobuf (in particular, because macros are not IDE friendly).